[Zaraz] Consent manager platform (#5746)
cloudflare/cloudflare-docs: content/zaraz/consent-management/_index.md

title: Consent management
weight: 4
meta:
    title: Zaraz Consent Management platform
layout: single
---

# Zaraz Consent Management platform

Zaraz provides a Consent Management platform (CMP) to help you address and manage required consents under the European [General Data Protection Regulation (GDPR)](https://gdpr-info.eu/) and the [Directive on privacy and electronic communications](https://eur-lex.europa.eu/legal-content/EN/TXT/HTML/?uri=CELEX:02002L0058-20091219&from=EN#tocId7). This consent platform lets you easily create a consent modal for your website based on the tools you have configured. With Zaraz CMP, you can make sure Zaraz only loads tools under the umbrella of the specific purposes your users have agreed to. 

The consent modal added to your website is concise and gives your users an easy way to opt-in to any purposes of data processing your tools need.

## Crucial vocabulary

The Zaraz Consent Management platform (CMP) has a **Purposes** section. This is where you will have to create purposes for the third-party tools your website uses. To better understand the terms involved in dealing with personal data, refer to these definitions:

> * **Purpose**: the reason you are loading a given tool on your website, such as to track conversions or improve your website’s layout based on behavior tracking. One purpose can be assigned to many tools, but one tool can be assigned only to one purpose.
> * **Consent**: this is an affirmative action that the user makes that is necessary in order to store and get access to cookies (or other persistent data, like `LocalStorage`) on the users’ computer/browser.

Reviewer: ```suggestion
* **Purpose**: The reason you are loading a given tool on your website, such as to track conversions or improve your website’s layout based on behavior tracking. One purpose can be assigned to many tools, but one tool can be assigned only to one purpose.
* **Consent**: An affirmative action that the user makes, required to store and access cookies (or other persistent data, like `LocalStorage`) on the users’ computer/browser.
```

{{<Aside type="note">}}

> All tools use consent as a legal basis. This is due to the fact that they all use cookies and are not strictly necessary for the website’s correct operation. Due to this, all purposes are opt-in.

Reviewer: ```suggestion
All tools use consent as a legal basis. This is due to the fact that they all use cookies that are not strictly necessary for the website’s correct operation. Due to this, all purposes are opt-in.
```
